kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
* regardless of window or element, we can't hard-code `600px` everywhere
* the side menu is an overlay at breakpoints `<=1`
* unless a component needs specific widths to fit its contents, using the global breakpoints helps everything switch "at the same time" (at the same width) which makes it easier to design specific mobile, tablet, and desktop behaviors